### DISCUSSION TITLE
Display embeds in editor mode

### DISCUSSION THREAD

**Discussion by joshmedeski** [MrGVSV/obsidian-github-embeds#1] [Ideas]
Hello, thanks for making this plugin! I love the way it looks but would really like to see the "Display embeds in editor mode" feature done in the near future.

I'm an experienced developer, can I help? Not sure how much work you've put into this yet @MrGVSV so let me know if you've done any work on this yet, thanks.

**Comment by MrGVSV** [comment]
Definitely! I haven't started that work yet, so if you'd like to give it a shot, feel free to make a PR! If not, I can try to prioritize it and get to it sometime in the next couple weeks.
